healthyeyes@ipolozov/files/healthyeyes@ipolozov/applet.js

```diff
@@ -24,19 +41,30 @@ MyApplet.prototype = {
   },
 
     refresh: function() {
-
+        //log("refresh!");
         let myDate = new Date();
+        let hour = myDate.getHours();
         let minute = myDate.getMinutes();
-
-        if (minute == 0 || minute == 30) {
+        
+        //log(hour);
+        //log(minute);
+        //log(last_hour);
+        //log(last_minute);
+	
+        // Checks if it should change, and makes sure that it isn't the same time as when it last changed
+        if ((minute == 0 || minute == 30) && (minute != last_minute || hour != last_hour)) {
+            log("change!");
```

Comment:
I wasn't sure if this was intentional or inadvertent. If it shouldn't be there, we can push the fix quickly.

Reply:
That log was meant to be left in, but if it's bad to have regular outputs then it could be removed

Reply:
It's up to you. Sometimes debug things slip past. 😅